README.md

```diff
@@ -69,38 +69,31 @@ Then, click on 'More'.
 
-Use `./bin/gisaid-get-all-clades` to perform this update.
-Python >= 3.6+, Node.js >= 12 (14 recommended) and yarn v1 are required.
+The most convenient option is to trigger it through the corresponding GitHub Action:
 
-```
-git clone https://github.com/nextstrain/ncov-ingest
-cd ncov-ingest
-yarn install
-pipenv sync
-BATCH_SIZE=1000 pipenv run ./bin/gisaid-get-all-clades
-```
-
-The resulting `data/gisaid/nextclade.tsv` should be placed on S3 bucked, replacing the one produced by the last daily ingest:
+ - [GISAID full Nextclade run](https://github.com/nextstrain/ncov-ingest/actions/workflows/nextclade-full-run-gisaid.yml)
+ - [GenBank full Nextclade run](https://github.com/nextstrain/ncov-ingest/actions/workflows/nextclade-full-run-genbank.yml)
 
-```
-./bin/upload-to-s3 data/gisaid/nextclade.tsv s3://nextstrain-ncov-private/nextclade.tsv.gz
-```
+They will simply run the `./bin/run-nextclade-full-aws --database=<name of the database>` and will announce the beginning of the job and the AWS Batch Job ID on Nextstrain Slack. 
 
-It will be picked up by the next ingest.
+For that, go to the GitHub Actions UI using one of the links above, click the button "Run workflow", choosing "branch: master" from the list and confirming.
 
-The best time for the update is between daily builds. There is usually no rush, even if the globally recomputed `nextclade.tsv` is lagging behind one or two days, it will be incrementally updated by the next daily ingest.
+If needed, the runs can be also launched from a local machine, by one of these scripts, depending on whether you want to run the computation locally, in docker, or to schedule an AWS Batch Job (the latter is what GitHub Actions do):
 
+```bash
+./bin/run-nextclade-full           # Runs locally (requires significant computational resources)
+./bin/run-nextclade-full-aws       # Runs in docker (requires significant computational resources)
+./bin/run-nextclade-full-docker    # Schedules an AWS Batch Job and runs there
+```
 
-## Required dependencies
-Run `pipenv sync` to setup an isolated Python 3.7 environment using the pinned dependencies.
+In case of AWS Batch option, the results of the computation, the new `nextclade.tsv` will be uploaded to S3 into a subdirectory in the directory which is the usual location of this file for the database. The subdirectory name will contain a date, so that there is no confusion about versions. The Slack announcement will contain the full path. These files then need to be manually inspected for correctness and scientific soundness and id all good, copied to the usual location where the daily ingest can find them. From that point the clades are considered fresh.
```

Comment:
```suggestion
With the AWS Batch option, the results of the computation – the new `nextclade.tsv` – will be uploaded to S3 into a subdirectory of the directory which is the usual location of this file for the database. The subdirectory name will contain a date, so that there is no confusion about versions. The Slack announcement will contain the full path. These files then need to be manually inspected for correctness and scientific soundness and if all good, copied to the usual location where the daily ingest can find them. From that point the clades are considered fresh.
```

Reply:
Thank you, sir :)